src/enums/currency.enum.ts

```diff
@@ -0,0 +1,3 @@
+export enum Currency {
+  $ = '$',
```

Comment:
save the abbriviation not the symbol
check here `https://www.easymarkets.com/int/learn-centre/discover-trading/currency-acronyms-and-abbreviations/`
so in this case itll be USD